pyproject.toml

```diff
@@ -33,21 +33,31 @@ dependencies = [
 
 [project.optional-dependencies]
 dev = [
-  'build',        # Building the package
-  'jinja2',       # Templating for the documentation
-  'nbmake',       # Building notebooks
-  'nbqa',         # Linting and formatting notebooks
-  'pytest',       # Testing
-  'pytest-cov',   # Coverage
-  'pytest-xdist', # Enable parallel testing
-  'ruff',         # Linting and formatting code
-  'versioningit', # Versioning
+  'build',                   # Building the package
+  'jinja2',                  # Templating for the documentation
+  'nbmake',                  # Building notebooks
+  'nbqa',                    # Linting and formatting notebooks
+  'pytest',                  # Testing
+  'pytest-cov',              # Coverage
+  'pytest-xdist',            # Enable parallel testing
+  'ruff',                    # Linting and formatting code
+  'validate-pyproject[all]', # Validate pyproject.toml
```

Comment:
Do we still need `npm` and `prettier` with `validate-pyproject`?

Reply:
I guess they are aimed at two different things - `validate-project` checks if the yml file is of valid syntax, while `prettier` checks/fixes the layout.

Reply:
Exactly. Plus, Prettier is used for all non-Python file types.